src/node.cc

```diff
@@ -630,6 +642,13 @@ inline void PlatformInit() {
   RegisterSignalHandler(SIGTERM, SignalExit, true);
 
 #if NODE_USE_V8_WASM_TRAP_HANDLER
+#if defined(_WIN32)
+  {
+    constexpr ULONG first = TRUE;
+    CHECK_NE(AddVectoredExceptionHandler(first, TrapWebAssemblyOrContinue),
+             nullptr);
```

Comment:
```suggestion
    CHECK_NOT_NULL(AddVectoredExceptionHandler(first, TrapWebAssemblyOrContinue));
```
Although "should be nullptr" is probably not a valid assumption for libnode as the embedder can already have registered exception handlers. I'd change it to this:
```c++
    // namespace per_process { PVOID old_vectored_exception_handler; }
    per_process::old_vectored_exception_handler =
        AddVectoredExceptionHandler(first, TrapWebAssemblyOrContinue);
```
And call `RemoveVectoredExceptionHandler()` in `TearDownOncePerProcess()`.

Reply:
what is `old_vectored_exception_handler` used for? do i pass it to `RemoveVectoredExceptionHandler`?